src/golang/lib/collections/operator/connector/load_params.go

```diff
@@ -37,8 +37,8 @@ type SalesforceLoadParams struct {
 }
 
 type S3LoadParams struct {
-	Filepath string `json:"filepath"`
-	Format   string `json:"format"`
+	Filepath string  `json:"filepath"`
+	Format   *string `json:"format"`
```

Comment:
Is the reason we need to change this to a pointer because we allow it to be None on the Python side? If so let's add a comment.

Reply:
yeah its because the pydantic model we expect to deserialize this into is Optional[enum], which was failing when getting the empty string.